typescript/infra/scripts/deploy.ts

```diff
@@ -111,7 +113,10 @@ async function main() {
     deployer = new TestQuerySenderDeployer(multiProvider);
   } else if (module === Modules.HELLO_WORLD) {
     const core = HyperlaneCore.fromEnvironment(env, multiProvider);
-    config = core.getRouterConfig(envConfig.owners);
+    config = objFilter(
+      core.getRouterConfig(envConfig.owners),
```

Comment:
I wonder if we should actually implement this filtering inside the `core.getRouterConfig`
the core implementation here is EVM specific

Reply:
not a bad shout - didn't want to do that initially in case it affected something else but if it's EVM specific anyway...